kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/dao/facade/CompanyFacade.java

```diff
@@ -29,31 +29,32 @@
     CompanyFacade() {
     }
 
-    public ResultDto search(String searched) throws CompanyFacadeException {
+    public ResultDto search(String searched) throws SearchException {
         if (StringUtils.isBlank(searched)) {
-            LOGGER.error(CompanyFacadeException.ERR_NULL_SEARCH);
-            throw new CompanyFacadeException(CompanyFacadeException.ERR_NULL_SEARCH);
+            LOGGER.error(SearchException.ERR_NULL_SEARCH);
```

Comment:
zastanawia mnie taka konsrukcja. Po co logujesz, że będzie wyjątek? Wyjątek powie sam za siebie (to raz), a ktoś będzie musiał ten wyjątek potem obsłużyć i on zaloguje błąd przecież.

Reply:
Zresztą moje pytanie: czy rzeczywiście ma rzucać metoda search wyjątkiem? Szukam czegoś i jak nie ma, to nie mówię, że coś jest nie tak, tylko że tego nie ma. A jak mam znaleźć nic, to znajduję nic przecież, czyli dla nulla zwracam null :)